UnitTests.Android/Moq.Android.Tests.csproj

```diff
@@ -231,9 +240,19 @@
       <Name>Moq.Android</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
```

Comment:
This txt file is added automatically by the NuGet package for xunit.runner.devices in case you don't have a MainActivity already. Please remove it.